lib/solr/params.rb

```diff
@@ -8,15 +8,17 @@ def initialize
       @parameters = {
         q: nil,
         qt: 'search',
-        df: 'keywords_teim',
+        df: 'keywords_teim', # TODO: Is this actually the default we want? This class is used by the TermSearchAdapter too.
         fq: [],
         'facet.field'.to_sym => [],
         start: 0
       }
     end
 
-    def fq(field, value)
-      @parameters[:fq] << "#{field}:\"#{Solr::Utils.escape(value)}\"" unless value.nil?
+    def fq(field, values, boolean_operator = :or)
```

Comment:
The array of all fqs are ANDed, but that boolean_operator param applies to values within a single fq.  In other words:

fq 1: something
fq 2: (something\ else OR another\ thing)

Reply:
I thought even within a fq we usually ANDed, but I think that might just be a blacklight default I am thinking about. Since we currently don't accept an array value for this method it wouldn't change any of the current placed were we use this method, which is what I was most worried about. I am okay with leaving as is.

Reply:
Okay cool, thanks.  If we end up doing a majority of ANDs within a single fq, I'm glad to change it.  I think on the blacklight side, those end up being separate fqs rather than being combined into a single one -- where the field is repeated in each separate fq.